### DISCUSSION TITLE
JDK更换为bellsoft的Liberica。

### DISCUSSION THREAD

**Discussion by moqsien** [gvcgo/version-manager#41] [Ideas]
https://bell-sw.com/pages/downloads/#jdk-21-lts

支持的比较完善，例如jdk1.8的苹果M芯片版本，以及JavaFX等。

@ikeukenhof

**Comment by ikeukenhof** [comment]
> https://bell-sw.com/pages/downloads/#jdk-21-lts
> 
> 支持的比较完善，例如jdk1.8的苹果M芯片版本，以及JavaFX等。
> 
> @ikeukenhof

我之前使用的是通过`brew`安装的[Azul Zulu OpenJDK](https://www.azul.com/downloads/?package=jdk#zulu)；
现在`vmr`还不具备下载`Liberica Standard JDK`或`Azul Zulu OpenJDK`的功能对吧？

**Reply by moqsien** [reply]
vmr只支持一种JDK发行版，挑个最好，覆盖面最广的就好。

一般来说，发行版对开发影响不大，就看哪个发行版覆盖得比较全了。

我看了一下Liberica，如果选择full版本的话，基本所有的都能覆盖到了。

这里的discussion，主要是vmr的贡献者或者有意向贡献代码的开发者讨论vmr相关功能设计和优化的地方。

这个JDK发行版来源的选择，可以提上议程。争取在简单易用的前提下，覆盖面足够广。

**Reply by ikeukenhof** [reply]
好的，期待更新🤩

**Reply by moqsien** [reply]
> 好的，期待更新🤩

0.6.1预览版已实现。

不过已经与更低版本不完全兼容了。如果感兴趣可以去看文档。
